demos/slider/multiple-vertical.html

```diff
@@ -5,11 +5,6 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>jQuery UI Slider - Multiple sliders</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/mouse.js"></script>
-	<script src="../../ui/slider.js"></script>
```

Comment:
Why doesn't this need to load JS anymore?

Reply:
It does still load JS. This was the old way of doing it that also included loading the deprecated core.js file that this PR removes.

This template actually used both ways of loading JS; it looks like a migration oversight. I checked that the demo still works fine after these removals.

Reply:
Good catch!